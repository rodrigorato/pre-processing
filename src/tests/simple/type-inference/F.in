public class Foo {
    public static void main(String[] args) {
		var a= new G();
		var b =new F();
		var c=new E();
		var 	 d= 	new D();
		var e = 
				new C();
		var 
			f = new B(); var 
			g = new A()		;
    }
}