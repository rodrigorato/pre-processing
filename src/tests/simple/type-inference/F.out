public class Foo {
    public static void main(String[] args) {
		G a= new G();
		F b =new F();
		E c=new E();
		D 	 d= 	new D();
		C e = 
				new C();
		B 
			f = new B(); A 
			g = new A()		;
    }
}